Source/PeleC.cpp

```diff
@@ -336,6 +336,13 @@ PeleC::read_params()
     amrex::Error("PeleC::ppm_type must be 1 (PPM) to use WENO method");
   }
 
+  if (do_hydro) {
+    if (do_mol || ppm_type == 0) {
+      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4))
+        amrex::Error("PeleC::plm_iorder must be 1, 2, or 4");
```

Comment:
I think for MOL we only really support 1 or 2, so there should be separate checks here for MOL and PLM

Reply:
good idea